modules/kms/variables.tf

```diff
@@ -15,16 +27,22 @@ variable "enable_key_rotation" {
   description = "Specifies whether key rotation is enabled"
 }
 
-variable "description" {
+variable "key_usage" {
   type        = string
-  default     = "Parameter Store KMS master key"
-  description = "The description of the key as viewed in AWS console"
+  default     = "ENCRYPT_DECRYPT"
+  description = "Specifies the intended use of the key. Valid values: `ENCRYPT_DECRYPT` or `SIGN_VERIFY`."
 }
 
-variable "alias" {
+variable "multi_region" {
+  type        = bool
+  default     = false
+  description = "Indicates whether the KMS key is a multi-Region (true) or regional (false) key."
+}
+
+variable "customer_master_key_spec" {
   type        = string
-  default     = ""
-  description = "The display name of the alias. The name must start with the word `alias` followed by a forward slash. If not specified, the alias name will be auto-generated."
+  default     = "SYMMETRIC_DEFAULT"
```

Comment:
Is this a breaking change or is this the normal default and we're just being explicit?